packages/extension-points/README.md

```diff
@@ -0,0 +1,36 @@
+# @drupal/extension-points-bundler
+
+Tooling to bundle and transpile your JavaScript to be used with the AdminUI.
+
+## Installation
+
+Add this module to your project. All the bundling and transpile dependencies and settings are included.
```

Comment:
"Add this npm module to your Drupal module"?

Reply:
yeah